PSMT/File.pm

```diff
@@ -408,11 +451,10 @@ sub ListDocsInPath {
     $sth->execute($pathid);
     my (@docid, @docs, $ref);
     while ($ref = $sth->fetchrow_hashref()) {push(@docid, $ref->{docid}); }
-    # cache
+    # call two for cache
```

Comment:
cache calls are better to be moved into GetDocsInfo (no foreach in this method now)